sapl/painel/views.py

```diff
@@ -368,17 +368,21 @@ def get_presentes(pk, response, materia):
     sessao = SessaoPlenaria.objects.get(id=pk)
     num_presentes = len(presentes)
     data_sessao = sessao.data_inicio
-    oradores = OradorExpediente.objects.filter(
-        sessao_plenaria_id=pk).order_by('numero_ordem')
-
+    
+    if type(materia) is ExpedienteMateria:
+        oradores = OradorExpediente.objects.filter(
+            sessao_plenaria_id=pk).order_by('numero_ordem')
+    else:
```

Comment:
```suggestion
    elsif isinstance(materia, OrdemDia):    
```

E ainda adicionaria um `else` colocando `oradores= []` (e logando que matéria não é de nenhum dos tipos suportados).

Reply:
Resolvido